Apps/Sandcastle/gallery/CZML Polygon - Intervals, Availability.html

```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE html>
```

Comment:
my intention was to show how to create time dynamic polygons using either reference properties & interpolation (CZML dynamic polygon - interpolation.html) or intervals & availability (this file), since according to the godfather of CZML, those were the two approaches. since this example is specifically the intervals/availability one and I'm also not able to reproduce the flicker, I'm not sure what other approach to take?